package.json

```diff
@@ -42,6 +42,7 @@
     "lodash": "^4.6.1",
     "resolve": "^1.1.7",
     "shelljs": "^0.6.0",
+    "multidep": "^2.0.0",
```

Comment:
This is used by broccoli 1.0 for installing additional plugins. Does not work without that for me.